README.md

```diff
@@ -108,75 +135,74 @@ Response (in Body):
   "contentType": "pdf",
   "created": "1970-01-01T00:00:00Z",
   "metadata": {},
-  "href": "/file-upload/envelopes/7989913d-b41f-4455-8baa-e8ed0168960c/files/file-id-1/content"
+  "href": "/file-upload/envelopes/0b215e97-11d4-4006-91db-c067e74fc653/files/file-id-1/content"
 }
 ```
 
 #### Download File
-Downloads a file from the envelope
+Downloads a file from an envelope.
 ```
 GET   	/file-upload/envelopes/{envelope-id}/files/{file-id}/content
 ```
 | Responses    | Status    | Description |
 | --------|---------|-------|
 | Ok  | 200   | Download a file.  |
 | Not Found | 404   |  File not found. |
-| Internal Server Error | 500   |  INTERNAL_SERVER_ERROR |
 
-Request (Get): localhost:8898/file-upload/envelopes/0b215e97-11d4-4006-91db-c067e74fc653/files/file-id-1/content
+Request (GET): localhost:8898/file-upload/envelopes/0b215e97-11d4-4006-91db-c067e74fc653/files/file-id-1/content
 
-Response: Download File
+Response: Downloads a binary file which contains the selected file.
 
 ### Routing
 
 #### Create File Routing Request
-Changes the status of an Envelope to CLOSED and rejects any requests to add files to the envelope. This makes it available for file transfer.
+Changes the status of an envelope to CLOSED, auto generates a routing Id. The status change, rejects any requests to add files to the envelope. This makes it available for file transfer. To make the request, the envelope Id, application and destination must be provided. In regards to MVP, only DMS is supported.
 ```
 POST    /file-routing/requests
 ```
 
 | Responses    | Status    | Description |
 | --------|---------|-------|
-| Created  | 201   | Successfully created event CLOSED |
-| Bad Request  | 400   |  Invalid Request. |
-| Bad Request  | 400   |  Destination not supported; Routing request already received for envelope |
-| Bad Request | 400   |  File contain errors; Envelope not found |
-| Forbidden | 403   |  Not Authorised. |
-| Internal Server Error  | 500   |  INTERNAL_SERVER_ERROR |  
+| Created  | 201   | Successfully created routing request.  |
+| Bad Request  | 400   | Failed to create route request. | 
 
 #### Example
 Request (POST): localhost:8898/file-routing/requests
 
 Body:
 ``` json
 {
-	"envelopeId":"8f494f63-d2e3-4ab1-9154-5f64a0b70f7c",
+	"envelopeId":"0b215e97-11d4-4006-91db-c067e74fc653",
 	"application":"application/json",
 	"destination":"DMS"
 }
 ```
 
-Response: 201
+Response(in Headers): Location -> /file-routing/requests/39e0e07d-7969-44ac-9f9c-4f7cc264b027
 
 ### Transfer
 
 #### Download List of Envelopes
-Retrieves a list of envelopes for routing. This applies to all envelopes that are closed.
+Returns either a list of all available or selected envelopes (via query string) for routing that have the status CLOSED and information from the routing request provided (above). 
 ```
 GET     /file-transfer/envelopes
```

Comment:
Actually after reading your 2 examples below it is explicit enough so please disregard this comment